### DISCUSSION TITLE
When we build docusaurus build, it creates a file as rumtime~main.[hash].js. The question is can we avoid ~?

### DISCUSSION THREAD

**Discussion by bharatmane** [facebook/docusaurus#10743] [General]
When I run docusaurus build
It generate the site as expected and it works very well. However in the build folder it generates few files, among them there is one file which is

**runtime~main.c20a87de.js**

The problem with this naming convention, when i try to deploy it to the server and browse the documentation. The search box is not functional and when i checked the why? I san an browser error in console

**_Failed to load resource: the server responded with a status of 500._**

This is because my server prohibits the name with tilt(~)

I tried multiple approaches to override but nothing seems working.

`module.exports = function forceWebpackRuntimeConfig() {
  return {
    name: 'force-webpack-runtime-config',
    configureWebpack(config, isServer) {
      // Explicitly override runtimeChunk
      config.optimization.runtimeChunk = {
        name: () => `runtime.[contenthash]`, // Generates runtime.[contenthash].js
      };

      // Override output filenames
      config.output.filename = '[name].[contenthash].js';
      config.output.chunkFilename = '[name].[contenthash].chunk.js';

      return config;
    },
  };
};`

I am trying to find if we can avoid the name being generated without tilt.
If you need more infromatio, please do let me know `

**Comment by slorber** [comment]
Tried this locally and this works:

```js
{
  optimization: {
    runtimeChunk: {
      name: (entrypoint) => `runtime___${entrypoint.name}`,
    },
  },
};
```

https://webpack.js.org/configuration/optimization/#optimizationruntimechunk

By default we use:

```js
    optimization: {
      runtimeChunk: true,
    },
```
